src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,21 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component adds previews of the top query results matching the currently focused query suggestion in the search box.
```

Comment:
Suggestion:

> This component renders previews to the top query results matching the currently focused query suggestion in the search box.

Reply:
Do you mean `to the top query results` or `of the top query results`? For now, I'll write `of`.

Reply:
Yes, _of_ is right, sorry :)